datalad_ria/tests/fixtures.py

```diff
@@ -79,3 +59,16 @@ def ria_sshserver(tmp_path_factory):
         info[e] = v
 
     yield info
+
+
+@pytest.fixture(autouse=False, scope="function")
+def ria_sshserver(ria_sshserver_setup, monkeypatch):
+    ria_baseurl = build_ria_url(
+        protocol='ssh',
+        host=ria_sshserver_setup['HOST'],
+        user=ria_sshserver_setup['SSH_LOGIN'],
+        path=ria_sshserver_setup['SSH_PATH'],
+    )
+    with monkeypatch.context() as m:
+        m.setenv("DATALAD_SSH_IDENTITYFILE", ria_sshserver_setup['SSH_SECKEY'])
+        yield ria_baseurl, ria_sshserver_setup['LOCALPATH']
```

Comment:
In the context of #58, to make the test run as it is currently implemented, I believe we need not only a ria_baseurl, but also a "host" to initialize the ``SSHRemoteIO`` with. Judging from its response when I hand it the riabaseurl, it wants to have it in one of two sytax:
```
Unsupported SSH URL: 'ria+ssh://adina@localhost/tmp/pytest-of-adina/pytest-33/sshriaroot0', use ssh://host/path or host:path syntax
```

Reply:
Yes and no. Yes, it needs it (best gotten from `ria_sshserver_setup` for now). And no, we do not necessarily want to continue using this abstraction.

However, it is unclear if and what exactly is the alternative, still.